### PR TITLE
push new links to seo component, remove langs checking domain

### PR DESCRIPTION
**Pull request by eduard-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by hasan-deriv — APPROVED** [review]

File: src/components/containers/seo.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { useStaticQuery, graphql } from 'gatsby'
 import { LocaleContext, localize } from '../localization'
 import language_config from '../../../i18n-config'
 import { isBrowser } from 'common/utility'
-import { eu_urls } from 'common/constants'
+import { eu_urls, hreflangLinks } from 'common/constants'
```
Comment (prince-deriv) [inline src/components/containers/seo.tsx:7]: let's follow the `snake_case` convention for arrays and variables.

File: src/components/containers/seo.tsx [hunk 2]
```diff
@@ -86,13 +86,19 @@ const SEO = ({
     let current_page = ''
     let organization_schema = {}
 
+    const languagesToRemove = site_url.includes('eu.deriv.com')
```
Comment (prince-deriv) [inline src/components/containers/seo.tsx:89]: same here

File: src/components/containers/seo.tsx [hunk 3]
```diff
@@ -86,13 +86,19 @@ const SEO = ({
...
+        ? ['th', 'vi', 'ko']
+        : ['it', 'pl', 'de'];
+
+    const filteredLanguages = languages.filter(lang => !languagesToRemove.includes(lang));
```
Comment (prince-deriv) [inline src/components/containers/seo.tsx:93]: here too

File: src/common/constants.ts [hunk 4]
```diff
@@ -185,3 +185,69 @@ export const bannerTypes = {
 export const firebaseConfig = {
     databaseURL: 'https://deriv-static-pricingfeed.firebaseio.com',
 }
+
+export const hreflangDerivLinks = [
```
Comment (habib-deriv) [inline src/common/constants.ts:189]:
```suggestion
export const hreflangRowLinks = [
```

**Review by Nuzhy-Deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**